app/templates/components/submission-status-cell.hbs

```diff
@@ -19,33 +20,34 @@
 </style>
 
 <div class="text-left">
-    {{#if (eq record.source 'other')}}
-      {{#if (eq record.submitted true)}}
-      <span class="status accepted pr-1">
+  <span class="status {{status}} pr-1">
+    {{#if (eq status "Complete")}}
+      {{#if (eq record.source "other")}}
         <span tooltip-position="left" tooltip="Funder has receive a submission for this publication from a different source.">
```

Comment:
This should be "All target repositories have received a copy of the Submission and verified that it is approved for publication"

Reply:
this looks fine too!

Reply:
Should `Complete` status care about the submission source?